schemas/database/database.md

```diff
@@ -0,0 +1,36 @@
+# Databases client spans
+
+Calls to databases should be tracked as client spans.
+
+Span `name` should be set to low cardinality value representing the statement
+executed on database. It may be stored procedure name (without argument), sql
+statement without variable arguments, etc. When it's impossible to get any
+meaningful representation of the span `name`, it can be populated using the same
+value as `db.instance`.
+
+**TODO: we might need have separate specs for other database types like Redis
```

Comment:
Maybe a TODO to add stats support for these calls. I would suggest to use stats for these request specific metrics. Maybe we also need an issue and explain when to use stats vs metrics

Reply:
I copied stats files. So this work will not be lost.